app/models/case_contact.rb

```diff
@@ -46,8 +74,9 @@ def humanized_type
 #
 # Indexes
 #
-#  index_case_contacts_on_casa_case_id  (casa_case_id)
-#  index_case_contacts_on_creator_id    (creator_id)
+#  index_case_contacts_on_casa_case_id   (casa_case_id)
+#  index_case_contacts_on_contact_types  (contact_types) USING gin
```

Comment:
gin?

Reply:
Updated with a comment on the migration!